app/controllers/comments_controller.rb

```diff
@@ -202,6 +203,61 @@ def preview
     end
   end
 
+  ## TODO: Service-orient this method - SubscriptionCreator.call(...)
+  # rubocop:disable Metrics/PerceivedComplexity, Metrics/CyclomaticComplexity
+  def subscribe
+    skip_authorization
```

Comment:
I'm curious why we're skipping authorization (and token verification) for this action? It seems like it needs to be authenticated (line 241, especially), but I'm possibly misunderstanding something.

Reply:
Seconded, I was curious about the auth skip here too